README.md

```diff
@@ -225,30 +225,21 @@ strategy](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Resolutio
 
 ### Commands
 
-The simplest way to publish a project and all its associated dependencies is to
-just publish all projects. The following command builds SNAPSHOT dependencies of
-all projects. All pom level dependencies within the published artifacts will
-also point to SNAPSHOT versions that are co-published.
+For more advanced use cases where developers wish to make changes to a project,
+but have transitive dependencies point to publicly released versions, individual
+projects may be published as follows.
 
 ```bash
-./gradlew publishAllToLocal
+# e.g. to publish Firestore and Functions
```

Comment:
Works fine for me, unless I'm missing something?

Reply:
If, for example, you want to test a change in `common` and `firestore`, you need to change the `common` dependency in `firestore` to project level dependency, otherwise the changes will not be visible.

For SDKs that have no relationship between them `publishAllToLocal` would work

Reply:
nvm, read the wrong side of the diff. thanks!